metacatalog/db/revisions/__init__.py

```diff
@@ -9,6 +9,7 @@
     rev7,
     rev8,
     rev9,
+    rev10,
```

Comment:
This can't be correct, there is already a revision 10, containing the current state of netCDF readers.